### PR TITLE
HODS-105 | Add endpoints, schemas, and examples to preliminary docs

### PR DESCRIPTION
**Pull request by Luka-Abey**

### CODE REVIEW

**Review by parkeradam — CHANGES_REQUESTED** [review]
Also seems to be mising Correlation Id Malformed erros and stuff, all sits in the sandbox-data section of the individuals repos.

**Review by parkeradam — APPROVED** [review]

**Review by ajchilts — CHANGES_REQUESTED** [review]

**Review by ajchilts — APPROVED** [review]

File: resources/public/api/conf/1.0/application.raml [hunk 1]
```diff
@@ -118,92 +119,92 @@ uses:
...
             required: false
         responses:
           200:
-            description: The company's Corporation Tax details
+            description: "The company's Corporation Tax details"
             body:
               application/json:
-                type: !include schemas/crn-details-response.json
-                example: !include examples/crn-details-response-example.json
+                type: !include schemas/paye-details-response.json
```
Comment (parkeradam) [inline resources/public/api/conf/1.0/application.raml:162]: thi payload seems wrong, running locally shows "taxpayer type and taxpayer name" whereas matrix shows employerName.

File: resources/public/api/conf/1.0/schemas/sautr-details-response.json [hunk 2]
```diff
@@ -1,32 +1,132 @@
...
+          "type": "string",
+          "description": "The first line of the organisation's address.",
+          "example": "123 Long Road"
+        },
+        "addressLine2": {
+          "type": "string",
+          "description": "The second line of the organisation's address.",
+          "example": "Some City"
+        },
+        "addressLine3": {
```
Comment (parkeradam) [inline resources/public/api/conf/1.0/schemas/sautr-details-response.json:41]: Missing description, check individuals as address descriptions have been solidified in there. Check addresse on all schemas in here please.

File: resources/public/api/conf/1.0/application.raml [hunk 3]
```diff
@@ -118,92 +119,92 @@ uses:
...
       get:
-        displayName: SAUTR Details
-        description: This endpoint takes a matchId and returns self assessment information for the organisation
+        displayName: "Get matched organisation's self-assessment details"
+        description: "This endpoint takes a matchId and returns self assessment information for the organisation"
         is: [headers.acceptHeader, headers.contentHeader]
         (annotations.scope): "assigned"
         securedBy: [ sec.oauth_2_0: { scopes: [ "assigned" ] } ]
+        (annotations.sandboxData): !include sandbox/sa-sandbox-data.md
         headers:
```
Comment (parkeradam) [inline resources/public/api/conf/1.0/application.raml:186]: Missing correlationId Header, should be on all endpoints.

**Review by parkeradam — CHANGES_REQUESTED** [review]

**Review by parkeradam — APPROVED** [review]

**Review by ajchilts — APPROVED** [review]

**Pull request merged**